packages/rs-drive/src/drive/grove_operations/grove_clear/v0/mod.rs

```diff
@@ -16,10 +17,35 @@ impl Drive {
             allow_deleting_subtrees: false,
             trying_to_clear_with_subtrees_returns_error: false,
         };
+
+        let maybe_path_for_logs = if tracing::event_enabled!(Level::TRACE) {
```

Comment:
why do you need this,  can't you just use `path=?&path` in line 42?

Reply:
The value is moved to `clear_subtree` function above so we need to clone. Since @QuantumExplorer hates unnecessary clone operations (even so small) I do the clone conditionally.